hal/src/utility.ts

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) Microsoft Corporation. All Rights Reserved. */
+
+import {hal} from './decorators';
+import {Rel, Hal} from './constants';
+import {Server} from './server';
+import {Response} from './response';
+import {Template} from './template';
+
+export function Utility(server: Object): Utility.Server {
```

Comment:
nit: you call it util all over the readme so i would use `Util` instead of Utility here